wuzz.go

```diff
@@ -832,7 +818,7 @@ func setViewTextAndCursor(v *gocui.View, s string) {
 	v.SetCursor(len(s), 0)
 }
 
-func quit(g *gocui.Gui, v *gocui.View) error {
+func quit(_ *gocui.Gui, _ *gocui.View) error {
```

Comment:
I personally don't see the value in eliding the argument names here, but that's a style call that @asciimoo should probably make the final decision on.